src/validation.h

```diff
@@ -105,17 +105,22 @@ static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 1000000;
 /** Additional block download timeout per parallel downloading peer (i.e. 5 min) */
 static const int64_t BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 500000;
 
+/** Default for -maxtipage, maximum tip age in seconds to consider node in initial block download. */
 static const int64_t DEFAULT_MAX_TIP_AGE = 24 * 60 * 60;
 /** Maximum age of our tip in seconds for us to be considered current for fee estimation */
 static const int64_t MAX_FEE_ESTIMATION_TIP_AGE = 3 * 60 * 60;
 
+/** Default for -checkpoints, enable skipping verification of known chain history until block 295000. */
```

Comment:
That's not what checkpoints do :slightly_smiling_face:

Reply:
To be fair the `-help-debug` was never updated either

Reply:
See:

* #16551

Reply:
Oh you're right, I've double-down on the error already there in the help-debug. Fixed both